scripts/test-prod.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+port=4873
+original_registry=`npm get registry`
+registry="http://localhost:$port"
+output="output.out"
+commit_to_revert="HEAD"
+test_app='rsp-cra-18'
+
+# Generate dists for the packages
+yarn parcel build packages/@adobe/react-spectrum/ packages/@react-{spectrum,aria,stately}/*/ packages/@internationalized/*/ --no-optimize --log-level error
+
+# Start verdaccio and send it to the background
+yarn verdaccio --listen $port &>${output}&
+
+# Login as test user
+yarn npm-cli-login -u abc -p abc -e 'abc@abc.com' -r $registry
```

Comment:
Any luck replacing this with the `npm add user` stuff I sent before? If not, I'm fine keeping this as is, we'll be modifying this script anyways when adding this to CI

Reply:
yeah unfortunately i haven't been able to get it to work